### ISSUE TITLE
SAE naming and docstring items deferred from #314 to avoid a GPU slot

### ISSUE THREAD

**Issue by speediedan** [speediedan/interpretune#315]
## Summary

#314 normalized SAE prose that meant *latent model* across `docs/` and `.github/`, and **deliberately excluded everything under `src/` and `tests/`** so it would consume no GPU slot. This issue captures what was excluded, so those items do not vanish into a merged PR body.

**Ride this along with another PR that already requires a GPU slot.** Every item below touches `src/` or `tests/`, both of which are in `.azure-pipelines/gpu-tests.yml`'s `paths.include`, so a standalone PR would spend a full ~40-minute serialized GPU run on what is otherwise a rename and two docstring corrections. There is no urgency justifying that.

## 1. Two docstrings document parameters that do not exist

Straight defects, independent of the SAE naming question, and the cheapest half of this issue.

**`LatentAnalysisDict.batch_join`** — `analysis/core.py:84`

```python
def batch_join(self, across_saes: bool = False, join_fn: Callable = torch.cat)
```

The docstring names **`join_across_saes`** three times (lines 90, 95, 96). No such parameter exists, here or in the protocol.

**`base_vs_sae_logit_diffs`** — `analysis/core.py:1184`

```python
def base_vs_sae_logit_diffs(sae, base_ref, tokenizer, top_k=10, max_prompt_width=80)
```

The docstring documents **`no_sae_ref`** (line 1195); the parameter is `base_ref`.

Both mislead anyone calling these by keyword. Fix independently of any renaming decision.

## 2. `across_saes` parameter name

A real parameter, not just prose, so renaming it to something latent-model-neutral (`across_latent_models`) is an API change with four sites:

```
src/interpretune/protocol.py:517          protocol signature
src/interpretune/analysis/core.py:85      definition
src/interpretune/analysis/core.py:98      internal use
src/interpretune/analysis/core.py:1260    call site: batch_join(across_saes=True)
```

Pre-MVP, so no alias per `CLAUDE.md`. **Decide whether this is worth doing at all** — unlike `logit_diffs_sae`, it is not user-facing in examples, though it is protocol surface.

## 3. `base_vs_sae_logit_diffs` is user-facing and named for SAEs

Exported from `interpretune.analysis` (`__init__.py:45,79`) and called in the **published** SAELens tutorial notebook. Renaming has the same shape and cost as #302:

| location | note |
|---|---|
| `analysis/__init__.py` | import + `__all__` |
| `analysis/core.py:1184` | definition, plus the `sae:` parameter name |
| `core.py:1225` | table header `"SAE Logit Diff"` |
| `notebooks/dev/` + `notebooks/publish/` saelens_adapter_example | 2 call sites each; **republish required** |
| `docs/notebook_artifacts/saelens_adapter_example` | 2 occurrences in captured output |
| `tests/core/test_analysis_core.py`, `test_model_backend_parity.py` | import + call sites |

**Genuinely arguable that this one should NOT change**: it compares a base run against a *latent-model* run, and in the tutorial that latent model is an SAE. Decide deliberately rather than by sweep momentum.

## 4. `...LogitDiffsSAE` config class names

**12 occurrences across 4 files**, targeting `logit_diffs_latent` since #308:

```
 6  tests/conftest.py
 3  tests/core/cfg_aliases.py          (definitions at :849, :952)
 2  .github/instructions/registering_example_modules.instructions.md
 1  tests/core/test_model_backend_parity.py   (TestLogitDiffsSAEBackendParity)
```

Note #314 **kept these names in the instructions doc on purpose** — they are real symbols, so renaming them in prose alone would have documented classes that do not exist. The doc and the code must move together.

## 5. Remaining "across SAEs" prose in `src/`

Consistency follow-through from #314, which normalized the same phrasing in `custom_ops_composition_guide.md`:

```
src/interpretune/analysis/core.py:87, 1255
src/interpretune/analysis/ops/bundled/composites.yaml:18
src/interpretune/analysis/ops/compiler/cache_manager.py:69
```

All describe `per_latent_preds`, whose field name is already neutral. **`cache_manager.py` is digest-guarded** (#293), so touching it needs the `EXPECTED_DIGESTS` pin updated — comment-only, so no `CACHE_FORMAT_VERSION` bump, but the guard will fire and the pin is the acknowledgement.

## Explicitly out of scope

`adapters/sae_lens.py`, `config/sae_lens.py`, `hook_mapping.py` and the NNsight backend impl all use SAE to mean an actual SAELens `SAE` object, and the sub-hook suffixes really are `hook_sae_acts_post`. Those stay.

## Suggested split

Items **1** (docstring defects) and **5** (prose) are trivial and uncontroversial — fold them into any `src/`-touching PR opportunistically.

Items **2**, **3** and **4** are renames needing a decision first, particularly **3**, where "keep" is defensible. They are not urgent and should not be bundled into an unrelated PR without that decision being made.

**Comment by speediedan** [comment]
Implemented in #319, riding #303's GPU slot exactly as this issue intended.

**Done:** both docstring defects (items 1); `across_saes` → `across_latent_models` (item 2); `...LogitDiffsSAE` → `...LogitDiffsLatent`, 12 occurrences across 4 files (item 4).

**Item 2 renamed** because the name contradicts its own context: the method is on `LatentAnalysisDict`, whose docstring is explicitly about latent model data, and its one call site reaches it through `by_latent_model(...)`. Not in any notebook, so protocol-surface only.

**Item 3 deliberately NOT renamed.** `base_vs_sae_logit_diffs` is exported from `interpretune.analysis` and called in a published tutorial, and unlike `across_saes` the name is **accurate where it is used**: it compares a base run against an SAE run, and `sae` receives SAE analysis in every in-tree call. Renaming user-facing surface deserves its own decision rather than a ride-along on a PR about requirement axes. Easy to override if you disagree, and cheaper than this issue currently suggests:

## Correction to this issue's cost model for item 3

> `docs/notebook_artifacts/saelens_adapter_example` | 2 occurrences in captured output

Measured across all three notebooks (dev, publish, docs artifact):

```
source=2   output=0    docs/notebook_artifacts/saelens_adapter_example/...ipynb
source=2   output=0    src/it_examples/notebooks/publish/saelens_adapter_example/...ipynb
source=2   output=0    src/it_examples/notebooks/dev/saelens_adapter_example/...ipynb
```

The symbol is in **source**, not captured output, in every one. That matters because it inverts the cost: an occurrence in captured output would require **re-executing** the notebook to rebuild the artifact, whereas source-only occurrences do not. So item 3 is **cheaper** than stated, and the current text discourages it for a reason that is not true.

I checked this expecting the opposite — I was going to argue that renaming would trip the output-drift gate added in #300. It would not.
